CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 * PR #1155: allow lazy materialization of query on load
 * Issue #1116: BigQuery write fails with MessageSize is too large
+* PR #1163: Added config to set the BigQuery Job timeout
```

Comment:
This should appear in the above line. We usually have all PR's listed first, and then all issues.

Reply:
Fixed